libs/core/src/components/pds-input/pds-input.scss

```diff
@@ -1,33 +1,30 @@
 :host {
-  --background: var(--pine-color-base-white);
-  --background-disabled: var(--pine-color-neutral-grey-100);
-
-  --color: var(--pine-color-neutral-charcoal-500);
+  --box-shadow-focus: 0 0 0 2px var(--pine-color-blue-200);
+  --box-shadow-focus-error: 0 0 0 2px var(--pine-color-red-200);
+
+  --color-text: var(-pine-color-charcoal-500);
```

Comment:
Should we add default here as well?

Reply:
same as above...not being used anywhere either

Reply:
All good!